Conference-Android/app/src/main/res/layout/conference_detail.xml

```diff
@@ -4,13 +4,15 @@
     xmlns:tools="http://schemas.android.com/tools"
```

Comment:
I am not sure what these changes actually did.  Can you describe what was the effect?